File: Modulo1/Semana7/ResolucaoPraticas/PI-024/Sources/Pessoa.cpp
#include "../Headers/Pessoa.hpp"

int Pessoa::getId(){
    return this->id;
}

string Pessoa::getCpf(){
    return this->cpf;
}

string Pessoa::getNome(){
    return this->nome;
}

void Pessoa::setId(const int &id){
    this->id = id;
}

void Pessoa::setNome(const string &nome){
    this->nome = nome;
}

void Pessoa::setCpf(const string &cpf){
    this->cpf = cpf;
}

File: Modulo2/JAVA/Semana4/ResoucaoPraticas/EmpresaTransporte/src/empresaTransporte/Pessoa.java
package empresaTransporte;

public abstract class Pessoa {
	
	private String nome;
	private String cpf;
	private int idade;
	private String RG;
	
	
	
	
	public Pessoa(String nome, String cpf, int idade, String rG) {
		super();
		this.nome = nome;
		this.cpf = cpf;
		this.idade = idade;
		RG = rG;
	}
	
	
	public String getNome() {
		return nome;
	}
	public void setNome(String nome) {
		this.nome = nome;
	}
	public String getCpf() {
		return cpf;
	}
	public void setCpf(String cpf) {
		this.cpf = cpf;
	}
	public int getIdade() {
		return idade;
	}
	public void setIdade(int idade) {
		this.idade = idade;
	}
	public String getRG() {
		return RG;
	}
	public void setRG(String rG) {
		RG = rG;
	}
	
	

}
